requirements/dev-requirements.txt

```diff
@@ -19,5 +19,6 @@ ipdb==0.9.4
 wheel==0.29.0
 graphviz==0.7
 git+git://github.com/nickpell/django-package-monitor@np/handle-no-semver
+transifex-client>=0.12.4
```

Comment:
worth pinning so we know the same thing's on everyone's machine?

Reply:
Hmm, I guess it IS scripted, so that might be wise.